app/javascript/src/components/ItemList/ItemListItemArchiveButton.jsx

```diff
@@ -0,0 +1,36 @@
+// Archive / unarchive button for item list
+
+// NOTE: although this element should clearly be a button and not an anchor,
```

Comment:
As I said in the PR description, this initially was a `<button>` when rendered inline, unfortunately as a child component (I also did try using plain HTML) it ignored all my attempts to use `preventDefault()`, `stopPropagation()`, `stopImmediatePropagation()`. The only way I could make it work was to transform this to an `<a>`.